frontend/ui/widget/textboxwidget.lua

```diff
@@ -151,6 +152,26 @@ function TextBoxWidget:init()
         self.text_height = self.lines_per_page * self.line_height_px
     end
 
+    self:_computeTextDimensions()
```

Comment:
Still, we're doing more than "computing text dimensions": we render the text...

And now that you're not even using this TextBoxWidget in FrontlightWidget, it feels unnecessary to hack this module and leave in there some code paths we're not even taking.

Reply:
That's kind of implied by the verb (same as _measireText) ;).

There's nothing really left of concern in init vs this, AFAICT.

And Button *can* take this path, so I'll take something that appeared to work just fine over a crash ;).

Reply:
> There's nothing really left of concern in init vs this

Except readability (not having to jump around to see what's happening) and messing my git history :)

> And Button can take this path, so I'll take something that appeared to work just fine over a crash 

Which you could have solved with a dumber setText() doing :free() + :init().